babi.py

```diff
@@ -914,7 +914,82 @@ def _get_char(stdscr: 'curses._CursesWindow') -> Key:
     return Key(wch, key, keyname)
 
 
-EditResult = enum.Enum('EditResult', 'EXIT NEXT PREV')
+EditResult = enum.Enum('EditResult', 'EXIT NEXT PREV EDIT')
+
+
+class Command:
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        keybinds: List[str],
+        func: Callable[[Screen, Key], EditResult],
+        aliases: List[str] = [],
+    ) -> None:
+        self.name = name
+        self.description = description
+        self.keybinds = keybinds
+        self.aliases = aliases
+        self.func = func
```

Comment:
this can probably be a `typing.NamedTuple`

Reply:
How would that work? I don't understand NamedTuples/Tuples well enough yet to get this

Reply:
Done.